plugins/BEdita/API/tests/TestCase/Controller/ObjectsControllerTest.php

```diff
@@ -628,13 +628,14 @@ public function testEdit()
         $this->assertResponseCode(200);
```

Comment:
I would add here and in `UsersControllerTest::testEdit()` a check on JSON response, redundant maybe...
```php
        $result = json_decode((string)$this->_response->getBody(), true);
        $this->assertEquals($data['id'], $result['data']['id']);
        $this->assertEquals($data['type'], $result['data']['type']);
        $this->assertEquals($data['attributes']['title'], $result['data']['attributes']['title']);
```